analyzers/src/SonarAnalyzer.CFG/Sonar/CSharpControlFlowGraph.cs

```diff
@@ -21,13 +21,12 @@
 using System;
 using System.Diagnostics;
 using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.CSharp;
 
 namespace SonarAnalyzer.CFG.Sonar
 {
     public static class CSharpControlFlowGraph
     {
-        public static bool TryGet(CSharpSyntaxNode node, SemanticModel semanticModel, out IControlFlowGraph cfg)
+        public static bool TryGet(SyntaxNode node, SemanticModel semanticModel, out IControlFlowGraph cfg)
```

Comment:
The `CSharpSyntaxNode` was an artificial limitation to enforce the contract. That we don't really need.